### ISSUE TITLE
Implement the external logins

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#47]
### Who

- [x] Anonymous visitor
- [x] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

Register using external IdPs.

### Why

To reuse existing accounts.

### Issue expands upon

This issue expands upon another issue(s): #21 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
